.vscode/settings.json

```diff
@@ -21,7 +21,8 @@
     "./netlify/edge-functions_tests",
     "./scripts/template/supabase",
     "./scripts/template/netlify-edge",
-    "netlify/edge-functions"
+    "netlify/edge-functions",
+    "./tests_backend"
   ],
-  "deno.importMap": ".netlify/edge-functions-import-map.json"
```

Comment:
We don't need this anymore?

Reply:
I missed this, I will add this back again in the latest commit

Thanks!